pa4/checker-pa4.ml

```diff
@@ -1212,121 +1241,101 @@ let main () = begin
         vertices := add_missing_edges !vertices;
         let sorted_classes = topo_sort !vertices in
 
-        (*List.iter (fun cls -> *)
-                (*printf "%s\n" cls*)
-        (*) sorted_classes ;*)
-
-        let does_method_exist_in_list target_method_name feature_list =
-                let result = ref false in
-                List.iter (fun feature ->
-                        match feature with
-                        | Attribute _ -> ()
-                        | Method ((_,method_name), _, _, _) -> begin
-                                if method_name = target_method_name then
-                                        result := true;
-                                        ()
-                        end
-                ) feature_list ;
-                !result
-        in
-
         let output_formal_name formals =
                 List.iter( fun ((_, formal_name),_) ->
                         fprintf f_out "%s\n" formal_name
                 ) formals                 
         in
 
+        let methods_map = ref (Hashtbl.create 255) in
```

Comment:
Are we using this variable anymore? It doesn't look like we're using this anymore. If we remove this variable we could make the variable in the get_parents_methods_tbl `methods_map` instead of `methods_map_`.